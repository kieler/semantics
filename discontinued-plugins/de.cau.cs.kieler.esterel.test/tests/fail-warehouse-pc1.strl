module warehouse_pc:

type MYSQL_RES_ptr;
type MYSQL_ROW;

function check_result(MYSQL_RES_ptr) : boolean;

input orders_results : MYSQL_RES_ptr;
output orders_query_out : string;  

signal item_quantity : integer,
       num_items_in_bin : integer,		 
       move_and_drop_item,
       robot_high_level_op_complete in
  if (check_result(?orders_results)) then
    emit num_items_in_bin(pre(?num_items_in_bin))
%   emit num_items_in_bin(pre(?num_items_in_bin)+?item_quantity);
  end if;
end signal

end module
