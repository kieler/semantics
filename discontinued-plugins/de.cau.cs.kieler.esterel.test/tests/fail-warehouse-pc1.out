16:pre not supported
